server/db.mjs

```diff
@@ -32,3 +23,53 @@ function initDb() {
 
   return pgp()(connection);
 }
+
+//TASK QUERIES
+export const getTasks = async (user) => {
+  let userId = parseInt(user);
+  const tasks = await db.any(`SELECT * FROM tasks WHERE "userId" = $1`, [
+    userId,
+  ]);
+  console.log({ tasks }); //TEST
+  return tasks;
+};
+
+export const addTask = async (
+  userId,
+  dueDate,
+  estTime,
+  summary,
+  description,
+) => {
+  let newTask = await db.any(
+    `INSERT INTO tasks("userId", "dueDate", "estTime", "summary", "description")
+VALUES($1, $2, $3, $4, $5) RETURNING id, summary`,
+    [userId, dueDate, estTime, summary, description],
+  );
+
+  return newTask;
+};
+
+export const deleteTask = async (id) => {
+  let deletedTask = await db.any(
+    `DELETE FROM tasks WHERE id = $1 RETURNING id, summary`,
+    [id],
+  );
+
+  console.log({ deletedTask });
+  return deletedTask;
+};
+
+//USER QUERIES
+export const getUser = async (email) => {
+  const user = await db.any("SELECT * FROM users WHERE email = $1", [email]);
+  if (user.length < 1) {
+    user = await db.any(
+      `INSERT INTO users("email")
+VALUES($1) RETURNING *`,
+      [email],
+    );
+  }
```

Comment:
Hmmmm, can you walk me through why a user is created here if "getting" the user is unsuccessful b/c no results were found?  I would expect this function to simply raise an error if no such user was found.

Reply:
Good question! This is for the edge case in which a person is logging into the app for the first time with their google account. Essentially, "signing up." Timedown's database will be storing data about the user that Google won't be storing, like their personal settings, tasks, and time blocks. It's particularly important if the user opts to not have their tasks automatically synced/added to their google calendar and only want them kept in the app. 

Right now I'm having it do this for logging in just for simplicity in testing, but I'll probably separate this into it's own "signing up" vs. "logging in" feature. Then, instead of throwing an error if no user is found, it'll throw an error if there is already a user with those credentials. :]